.github/workflows/build-snapshot-worker.yml

```diff
@@ -60,7 +60,7 @@ jobs:
         jfrog rt mvn -U -B clean install -T 0.5C
         jfrog rt build-publish
         echo BUILD_ZOO_HANDLER_spring_cloud_deployer_version=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout) >> $GITHUB_ENV
-        echo BUILD_ZOO_HANDLER_spring_cloud_deployer_buildname=spring-cloud-deployer-main >> $GITHUB_ENV
+        echo BUILD_ZOO_HANDLER_spring_cloud_deployer_buildname=spring-cloud-deployer-main-3 >> $GITHUB_ENV
```

Comment:
When we merge to main branch we need to reset these back to main.

Reply:
Yep - good catch. We will do a pass through all the workflows after the merge. This one did not come up during the rebase/conflicts so I missed it.